### PR TITLE
update documentation and code in preparation for ERTP 0.1.0 (released)

### PR DESCRIPTION
**Pull request by katelynsills**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,40 +96,94 @@ The HTML frontend code is pure JS/DOM (no additional libraries yet), in
...
+```
+
+You might see something like: 
+
+```js
+{"label":{"issuer":[Presence 15],"description":"pixels"},"quantity":[{"x":1,"y":4}]}
+```
+
+The `quantity` tells us which pixels we've received. `{ x:1, y:4 }`
+means that we got a pixel that is in the second row (`x:1`) and 5 pixels
```
Comment (michaelfig) [inline README.md]: I think this is x/y swapped.

Comment (katelynsills) [reply README.md]: I think you're right that it's swapped, but I think the mistake is in the front-end code, so this documentation matches what currently happens. I turned this pixel {x:1, y:4} black:

Screen Shot 2019-08-13 at 9 46 27 PM

File: README.md [hunk 2]
```diff
@@ -96,40 +96,94 @@ The HTML frontend code is pure JS/DOM (no additional libraries yet), in
...
+methods that allow you to obtain a few pixels for free, color them,
+sell them, and buy more.
 
 To access the gallery, type `home.gallery` in the REPL. `home.gallery` is a
-Presence. In the SwingSet environment, Presences are remote references to objects on
-other vats. To invoke them, use the [proposed infix bang](https://github.com/Agoric/proposal-infix-bang). For example, the
+a remote object (what we call a *presence*). It actually lives in another environment (what we
+call a *vat*). We can call methods on remote objects as if the objects were
+local by replacing the dot syntax (`obj.foo`) with a [bang](https://github.com/Agoric/proposal-infix-bang) (`obj!foo`). For example, the
 first thing you might want to do is tap the gallery faucet to get a
```
[the remark this answers is not in the document]
Comment (katelynsills) [reply README.md]: Good point. I think I fixed it just now

[the remark this answers is not in the document]
Comment (erights) [reply README.md]: Looks good

**Pull request merged**